package.json

```diff
@@ -2,22 +2,24 @@
   "name": "eslint-find-rules",
   "version": "0.0.0-semantically-released",
   "description": "Find built-in ESLint rules you don't have in your custom config.",
-  "main": "src/lib/rule-finder.js",
+  "main": "dist/lib/rule-finder.js",
   "scripts": {
     "cover": "nyc --reporter=lcov --reporter=text npm test",
+    "lint": "xo",
     "test": "mocha --recursive",
-    "lint": "eslint --ignore-pattern test/fixtures .",
     "update-contributors": "all-contributors generate",
     "commit": "git-cz",
     "validate": "npm-run-all --parallel lint cover --sequential check-coverage",
     "check-coverage": "nyc check-coverage --statements 100 --branches 100 --functions 100 --lines 100",
+    "build": "babel src -d dist --presets es2015",
     "report-coverage": "cat ./coverage/lcov.info | node_modules/.bin/codecov",
     "semantic-release": "semantic-release pre && npm publish && semantic-release post",
     "travis-after-all": "travis-after-all && npm run report-coverage && npm run semantic-release"
   },
+  "files": ["dist"],
```

Comment:
Please do not ever use `files` - it's incredibly dangerous, because if you forget to add a file to this whitelist, the consequence is that the package is broken in a way that's very hard to detect.

It's fine to use `.npmignore` if you need to exclude files - but I also don't see any reason to exclude any of them. `npm explore foo && npm install && npm test` should always work.